install/kubernetes/cilium/templates/clustermesh-apiserver-clusterrole.yaml

```diff
@@ -4,12 +4,6 @@ kind: ClusterRole
 metadata:
   name: clustermesh-apiserver
 rules:
-- apiGroups:
```

Comment:
This particular patch seems a bit odd. The upstream commit reference is missing, and the original commit is very different. It would be nice to include some minimal commentary in the commit message as to why the backports differ (doesn't necessarily have to get into deep detail, just signalling that the patch is _deliberately_ very different from the original rather than unintentionally different).